test/unit/MicroserviceErrorHandler.test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require('chai');
+
+const MicroserviceErrorHandler = require('../../src/MicroserviceErrorHandler');
+
+describe('The MicroserviceErrorHandler class', function(){
+    it('can be instantiated with options containing the serviceName', function(){
+        const serviceName = 'unit-test-service';
+        const handler = new MicroserviceErrorHandler({ serviceName });
+
+        expect(handler).to.have.property('serviceName', serviceName);
+    });
+
+    it('can be instantiated with options containing the microservice config object', function(){
+        const microservice = {
+            name: 'unit-test-service',
+        };
+        const handler = new MicroserviceErrorHandler({ microservice });
+        expect(handler).to.have.property('serviceName', microservice.name);
+    });
+
+    it('throws and error if it cannot extract a service name (there is no reason to use it otherwise' +
+        ' at this time', function(){
+        expect(() => new MicroserviceErrorHandler({})).to.throw();
```

Comment:
May you should check what the ErrorHandler throws.

Reply:
Basically I don't care ;) I just want the application to not be able to boot.